bin/common.sh

```diff
@@ -6,23 +6,36 @@ readonly clustersFile="$ENV_DIR/env/clusters.yaml"
 readonly helmfileOutputHide="(^\W+$|skipping|basePath=|Decrypting)"
 readonly helmfileOutputHideTpl="(^[\W^-]+$|skipping|basePath=|Decrypting)"
 readonly replacePathsPattern="s@../env@${ENV_DIR}@g"
+readonly yqDockerImage="mikefarah/yq:3"
+
+function dyq() {
+  local yqcommand=$@
+  if [ -x "$(command -v yq)" ] && [[ "$(yq --version 2>&1 | sed 's/yq version //g')" == "3."* ]] ; then
```

Comment:
This conditional statement may need some changes. As far I understand the requirements, we just need to check if we are running in docker context or not.
1) If docker context then `yq` should already exist as a part of the `otomi-core` docker image. 
       if yq does not exist then return error
2) If otomi command is run on a regular host then run dockerized version of `yq`.

Reply:
That might be true, but lets assume:

1. in docker context yq exists, so first check should pass and the "yq" should run
2. On regular host
    a. yq is installed on version 3: Why not use local yq, it's more performant and no harm's done
    b.  yq is not installed (on version 3): Try to run dockerized version of yq
    
The second part of the conditional statement just makes sure that yq is running in V3. In the otomi-core image that's true, so will pass. On local machine fail if yq@4 is installed, then try dockerized yq@3

Or is it an incorrect assumption?

Reply:
2) On host yq can have different versions that the one we are using in docker file
Maybe some have yq@3.0.0 which has some bugs and we did not tested it.
As you can see we are not able to control user binaries. Moreover we do not want to enforce a user to install any binary except docker. So I think is wise to use always the same `yq`.

Reply:
Changed implementation, so marking this as resolved for now